README.md

```diff
@@ -0,0 +1,143 @@
+# FP-Signature
+
+Fynd platform's signature logic
+
+
+
+FP-Signature is a versatile npm package that provides signature logic for Fynd platform requests. This package supports both CommonJS and ES modules, and it also comes with a web bundle for direct usage in browsers.
+
+## Installation
+
+You can install FP-Signature via npm:
+
+```bash
+npm install @gofynd/fp-signature
+```
+
+## How to Use
+
+### For CommonJS
+
+```javascript
+const { sign } = require("@gofynd/fp-signature");
+```
+
+### For ES Modules
+
+```javascript
+import { sign } from "@gofynd/fp-signature";
+```
+
+### For Browser
+
+```html
+<script src="https://cdn.jsdelivr.net/npm/@gofynd/fp-signature@{version}"></script>
+<script>
+    // FP-Signature library will be attached to the global window object
+    FPSignature.sign();
+</script>
+```
+
+## Functionality
+
+### `sign` function
+
+The `sign` function is used to generate a signature. It takes two parameters: `request` and `options` and returns `x-fp-date` and `x-fp-signature`.
+
+
+```typescript
+type RequestParam = {
+  method: string;
+  host?: string;
+  port?: number;
+  path?: string;
+  headers?: any;
+  body?: any;
+  doNotEncodePath?: boolean;
+  doNotModifyHeaders?: boolean; 
+};
+
+type SigningOptions = {
+    secret?: string;
+    forQuery?: boolean;
```

Comment:
change this option to signQuery @bhargavprajapatiFynd

Reply:
Changed the name